client/src/views/PasswordReset.jsx

```diff
@@ -1,9 +1,214 @@
+    "https://images.pexels.com/photos/389818/pexels-photo-389818.jpeg"
+  );
 
-export default PasswordReset
\ No newline at end of file
+  const navigate = useNavigate("");
+  const theme = useTheme();
+  const colors = tokens(theme.palette.mode);
+  const colorTheme = useColorTheme();
+  const { currentUser, setCurrentUser, setCurrentUserEmail } =
+    useContext(AuthContext);
+
+  const getRandIdx = () => {
+    return Math.floor(Math.random() * imgArr.length);
+  };
+
+  useEffect(() => {
+    // Define an asynchronous function to fetch images from the Pexels API
+    const getImage = async (image = "house%20interiors", numImage = 15) => {
+      // Construct the API URL with the provided image query and per_page parameter.
+      const url = `https://api.pexels.com/v1/search?query=${image}&per_page=${numImage}`;
+      // Make an HTTP GET request to the Pexels API.
+      // try catch block so if the fetch method fails we can fix it
+      try {
+        const response = await fetch(url, {
+          method: "GET",
+          headers: {
+            // Set the Authorization header with the Pexels API key from environment variables.
+            Authorization: process.env.REACT_APP_PEXELS_API_KEY,
+            "Content-Type": "application/json",
+          },
+        });
+
+        if (response.status === 200) {
+          const data = await response.json();
+          setImgArr(data.photos);
+        } else {
+          console.error(await response.json());
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+    // Call the getImage function immediately when the component mounts.
+    getImage();
+    // The empty dependency array [] ensures this effect runs only on component mount and unmount.
+  }, []);
+
+  useEffect(() => {
+    if (imgArr.length > 0) {
+      //declaring the interval
+      const interval = setInterval(() => {
+        //once this times out do it again
+        // new Idx for image (which is a random #)
+        let newIdx = getRandIdx();
+        // if thant new num == img we're already on, get a new num till its not the same
+        while (newIdx === bgImageIdx) {
+          newIdx = getRandIdx();
+        }
+        //set the new num
+        setBgImageIdx(newIdx);
+        setTimeout(() => {
+          setBgImageIdx(newIdx);
+          setBgImgString(imgArr[newIdx].src.original);
+        }, 750);
+      }, 5000);
+      // clear interval so that we start a new interval and changes the background image
+      return () => clearInterval(interval);
+    } else {
+      // console.log("No Images to cycle")
+    }
+  }, [bgImageIdx, imgArr]);
+
+  const handleReset = (e) => {
+    e.preventDefault();
+    setEmailError(false);
+
+    if (!email || !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(email)) {
+      setEmailError(true);
+      return;
+    }
+    console.log(sendPasswordResetEmail(auth, email));
+    sendPasswordResetEmail(auth, email)
```

Comment:
I see this method imported above.  Looks pretty standard but always doublecheck the docs for firebase on this

Reply:
will do

Reply:
Double checked docs for sendPasswordResetEmail implementation